site/learn/expert-advice/tokens/jwt-components-explained.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: advice
-title: Components of JWT Authentication Tokens Explained
-description: Technical authentication article explains JSON Web Tokens (JWT) and their component parts. 
+title: Components of JWTs Explained
```

Comment:
Also need update the title on the brochure site.

learn/expert-advice/tokens/index.html:14

```
{% include _advice-card.liquid href="/learn/expert-advice/tokens/jwt-authentication-token-components-explained" img="/assets/img/stamps/anatomy-of-a-jwt-stamp.svg" title="Components of JWT Authentication Tokens Explained" description="JSON Web Tokens (JWTs) look like gibberish. But they are actually quite structured, with a header, payload and signature." %}
```